CAE/CorporateEvents/CorporateActions.rdf

```diff
@@ -182,31 +237,44 @@
+		<skos:note xml:lang="en">Divestiture is an action or process involving the sale, spinoff, or liquidation of business assets, including product lines, services, subsidiaries, business property, or even an entire business. Companies may pursue a divestment strategy to refocus on their core business, in response to the operating environment in their industry, to raise capital, or to release underperforming assets.</skos:note>
+	</owl:Class>
+	
+	<owl:Class rdf:about="&fibo-cae-ce-act;IncomeOrientedClassifier">
+		<rdfs:subClassOf rdf:resource="&fibo-cae-ce-act;ActionClassifier"/>
+		<rdfs:label xml:lang="en">income-oriented classifier</rdfs:label>
+		<skos:definition xml:lang="en">classifier of corporate actions that impacts income to shareholders</skos:definition>
+		<fibo-fnd-utl-av:explanatoryNote xml:lang="en">Cash dividends are a classic example where a public company declares a dividend to be paid on each outstanding share. Bonus is another case where the shareholder is rewarded. In a stricter sense, the bonus issue should not impact the share price but in reality, in rare cases, it does and results in an overall increase in value.</fibo-fnd-utl-av:explanatoryNote>
 	</owl:Class>
 	
 	<owl:Class rdf:about="&fibo-cae-ce-act;LegalFormChange">
 		<rdfs:subClassOf rdf:resource="&fibo-cae-ce-act;CorporateInformationAction"/>
 		<rdfs:label xml:lang="en">legal form change</rdfs:label>
 		<skos:definition xml:lang="en">corporate action that informs stakeholders of a modification of the legal form of the organization</skos:definition>
-		<skos:example xml:lang="en">In the United States it is common for companies established as Subchapter S Corporations (S-Corp), typically early stage companies, modify their structure to become full-fledged Subchapter C Corporations (C-Corp) to facilitate outside fundraising, mergers, acquisitions, and public offerings.  Other common form changes include migration from sole proprietorships to more formally registered organizations (e.g., LLC, S-Corp, C-Corp, etc.)</skos:example>
+		<skos:example xml:lang="en">In the United States it is common for companies established as Subchapter S Corporations (S-Corp), typically early stage companies, to modify their structure to become full-fledged Subchapter C Corporations (C-Corp) to facilitate outside fundraising, mergers, acquisitions, and public offerings.  Other common form changes include migration from sole proprietorships to more formally registered organizations (e.g., LLC, S-Corp, C-Corp, etc.)</skos:example>
 	</owl:Class>
 	
 	<owl:Class rdf:about="&fibo-cae-ce-act;LiquidationAction">
-		<rdfs:subClassOf rdf:resource="&fibo-cae-ce-act;MandatoryCorporateAction"/>
+		<rdfs:subClassOf rdf:resource="&fibo-cae-ce-act;DivestitureAction"/>
 		<rdfs:label xml:lang="en">liquidation action</rdfs:label>
 		<skos:definition xml:lang="en">corporate action consisting of distribution of cash, assets, or both</skos:definition>
```

Comment:
Yes but this liquidation action also covers cases where from a business optimization perspective that the business is selling assets vs. ultimate liquidation

Reply:
Modified the definition to specifically state that liquidation is the winding up of a business, and other cases should be individuals of type divestiture rather than of type liquidation, such as the liquidation of a product line or subsidiary